test-utils/testlib/src/node/process_node.rs

```diff
@@ -47,6 +47,7 @@ impl Node for ProcessNode {
     fn start(&mut self) {
         match self.state {
             ProcessNodeState::Stopped => {
+                std::env::set_var("ADVERSARY_CONSENT", "1");
```

Comment:
What is this for?

Reply:
nayduck builds the binary with adversarial features, this is required to run it